### DISCUSSION TITLE
下载forge失败  请求的操作需要提升

### DISCUSSION THREAD

**Discussion by Sxiang51390** [Meloong-Git/PCL#5000] [提问] [answered]
下载 Forge 40.2.21失败
→ 安装 Forge（方式 A）失败
→ 安装新 Forge 版本失败
→ 运行 Forge 安装器失败
→ 请求的操作需要提升。
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__13-0._Lambda$__0()
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)
   在 PCL.ModDownloadLib.ForgelikeInjector(String Target, LoaderTask`2 Task, String McFolder, Boolean UseJavaWrapper, Boolean IsNeoForge)
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)
错误类型：System.ComponentModel.Win32Exception

**Comment by Sxiang51390** [comment]
起因是把游戏启动进程设置为管理员身份运行 导致游戏无法启动

**Comment by LTCatt** [comment, marked as answer]
如果现在没用管理员权限就用，如果现在在用就不用……
